src/components/Table/Table.tsx

```diff
@@ -0,0 +1,142 @@
+import React, { useState, useEffect } from 'react';
+import { Person } from '../../types';
+import { useParams } from 'react-router-dom';
+import { Loader } from '../Loader';
+import { getPeople } from '../../api';
+
+export const Table: React.FC = () => {
+  const { personSlug } = useParams();
+  const [requestState, setRequestState] = useState('loading');
```

Comment:
It's not a great idea to store hard-coded strings like this.
I understand that you wanted to avoid multiple _useState_ declarations and decided to track _errors_, _loading_ and _empty_ in the same state.
However, it really isn't necessary as there is no need to store _empty_ in the state.
The second point is, if you really want to keep your state like this, please move these strings to an _Enum_, to guarantee it will be restricted to the possible cases

Reply:
I think a diff way bcz tests of mateacademy scripts but I'll work for put your style code here, is possible to change and improve this accordly your style code and advices.